scripts/qa_crowbarsetup.sh

```diff
@@ -2536,6 +2537,16 @@ function oncontroller_testsetup()
     . .openrc
 
     export LC_ALL=C
+
+    # prepare test image with the -test packages containing functional tests
+    if iscloudver 6plus; then
+        local mount_dir="/tmp/Cloud-Testing"
```

Comment:
@dirkmueller already merged. Pardon. But created another PR to fix this: https://github.com/SUSE-Cloud/automation/pull/525